lib/ui/text/paragraph.cc

```diff
@@ -52,42 +52,52 @@ size_t Paragraph::GetAllocationSize() {
 }
 
 double Paragraph::width() {
+  UIDartState::ThrowIfUIOperationsProhited();
```

Comment:
This seems redundant. Methods invocations on this C++ object should already go through `FOR_EACH_BINDING` to `DART_NATIVE_CALLBACK` which you have already guarded below.

Reply:
Ah, true. Removed.